internal/alert/routing.go

```diff
@@ -22,6 +24,7 @@ type RoutingDirectory interface {
 type routingDirectory struct {
 	pagerDutyClients map[core.Severity][]client.PagerDutyClient
 	slackClients     map[core.Severity][]client.SlackClient
+	snsClient        client.SNSClient
```

Comment:
Lets comment why this deviates from its above field counterparts

Reply:
[74f2414](https://github.com/base-org/pessimism/pull/200/commits/74f241413764e3b9caf17a604ffe27f2bc28480c)